include/swift/Frontend/InputFile.h

```diff
@@ -63,15 +68,33 @@ class InputFile {
 
   /// Return Swift-standard file name from a buffer name set by
   /// llvm::MemoryBuffer::getFileOrSTDIN, which uses "<stdin>" instead of "-".
-  static StringRef convertBufferNameFromLLVM_getFileOrSTDIN_toSwiftConventions(
+  static std::string
```

Comment:
I think it's okay for this one to stay StringRef to avoid copies. Yes, it means that the input can't be an expiring std::string, but that's pretty unlikely in practice, and ASan would catch it.

Reply:
I agree, tx for catching this. Fixed.